File: src/main/Java/boj_step5/boj9086.java
import java.io.*;

public class boj9086 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int t = Integer.parseInt(reader.readLine());
        while (t-- > 0) {
            String s = reader.readLine();
            writer.append(s.charAt(0) + "" + s.charAt(s.length() - 1));
            writer.newLine();
        }
        writer.close();
    }
}

File: src/main/Kotlin/others/boj9086.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    repeat(n) {
        val s = r.readLine()!!
        w.write(s.first() + s.last().toString() + "\n")
    }
    w.close()
}